toolkit/sidechain/domain/src/lib.rs

```diff
@@ -797,27 +870,41 @@ impl Display for McBlockHash {
 	}
 }
 
+/// Extended information about an (unspent) transaction output in Cardano ledger
```

Comment:
```suggestion
/// Extended information about a UTxO in Cardano ledger
```
UTxOs are explained above and should be assumed to be known by anyone using this library as it is a fundamental term.